src/test/objectstore/store_test.cc

```diff
@@ -903,6 +903,205 @@ TEST_P(StoreTest, CompressionTest) {
   g_ceph_context->_conf->apply_changes(NULL);
 }
 
+TEST_P(StoreTest, garbageCollection) {
```

Comment:
This UT verifies almost nothing - there is neither checks that GC has been executed and produced any results, nor data consistency after its' application. It will successfully pass without any GC functionality as well hence IMO it has  little sense.